OpenRA.Mods.RA/Effects/GpsSatellite.cs

```diff
@@ -36,8 +41,12 @@ public void Tick(World world)
 			anim.Tick();
 			pos += new WVec(0, 0, 427);
 
-			if (pos.Z > pos.Y)
+			if (++tick >= revealDelay)
+			{
+				var owner = launcher.Owner.PlayerActor.Trait<GpsWatcher>();
```

Comment:
If I'm not mistaken, this will change the behaviour subtly in the case where an engineer captures an atek while the GPS satellite is being launched. In this PR, the new owner will get the GPS, previously the old owner would have gotten it. Moving this line up into the constructor would restore the old behaviour.